### PR TITLE
examples: Add TDX-based oracle

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: client-sdk/ts-web/rt/playground/sample-run-network.sh [hunk 1]
```diff
@@ -38,7 +40,55 @@ jq '
...
+# Run test runner to generate genesis document.
 "$TEST_NET_RUNNER" \
     --fixture.file "$FIXTURE_FILE" \
-    --basedir /tmp/oasis-net-runner-sdk-rt \
-    --basedir.no_temp_dir
+    --basedir "$WORKDIR" \
+    --basedir.no_temp_dir &
+RUNNER_PID=$!
+
+# Below is a workaround for there being no way to change the default max tx size which
```
Comment (ptrus) [inline client-sdk/ts-web/rt/playground/sample-run-network.sh:50]: Do we have an issue in oasis-core, for adding support to this vis a fixture?

**Review by ptrus — APPROVED** [review]

File: client-sdk/ts-web/rt/playground/sample-run-network.sh (hunk 1 above)
Comment (kostko) [reply client-sdk/ts-web/rt/playground/sample-run-network.sh:50]: Good point, I'll open an issue. Core otherwise has the increased max tx size in master.

Comment (kostko) [reply client-sdk/ts-web/rt/playground/sample-run-network.sh:50]: Done, https://github.com/oasisprotocol/oasis-core/issues/5960.

Comment (ptrus) [reply client-sdk/ts-web/rt/playground/sample-run-network.sh:50]: Thx!

**Pull request merged**